workflow/setup_expt.py

```diff
@@ -144,6 +149,19 @@ def link_files_from_src_to_dst(src_dir, dst_dir):
         makedirs_if_missing(dst_dir)
         link_files_from_src_to_dst(src_dir, dst_dir)
 
+    # Link aerosol files
+    if do_aerosols:
+        # TODO: do not know what warm start ICs for aerosols will look like
```

Comment:
they are the same as the ATM RESTART files, `fv_tracer`, so no need to do anything special here I don't think

Reply:
What Cory said. Aerosol tracers are included in the regular atmosphere restart files.